release-notes/8.0/preview/api-diff/preview4/Microsoft.NETCore.App/8.0-preview4_System.Runtime.InteropServices.md

```diff
@@ -0,0 +1,38 @@
+# System.Runtime.InteropServices
+
+``` diff
+ namespace System.Runtime.InteropServices {
+     public static class CollectionsMarshal {
++        public static void SetCount<T>(List<T> list, int count);
+     }
+     public class COMException : ExternalException {
+         protected COMException(SerializationInfo info, StreamingContext context);
+     }
+     public class ExternalException : SystemException {
+         protected ExternalException(SerializationInfo info, StreamingContext context);
+     }
+     public class InvalidComObjectException : SystemException {
+         protected InvalidComObjectException(SerializationInfo info, StreamingContext context);
+     }
+     public class InvalidOleVariantTypeException : SystemException {
+         protected InvalidOleVariantTypeException(SerializationInfo info, StreamingContext context);
+     }
```

Comment:
These changes are deprecating the `protected` constructors, not adding them. This was done in https://github.com/dotnet/runtime/pull/84383.